docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,398 @@
+      "text/plain": [
+       "<Figure size 1200x500 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "soiling = kimber(\n",
+    "    rainfall_data.precipitation_rate, \n",
+    "    cleaning_threshold=15.0, \n",
+    "    grace_period=3, # Number of days after a rainfall event when it’s assumed the ground is damp, and so it’s assumed there is no soiling. \n",
+    "    soiling_loss_rate=0.0015 # Fraction of energy lost due to one day of soiling.\n",
+    ")\n",
+    "\n",
+    "df = pd.concat([rainfall_data, soiling], axis=1)\n",
+    "\n",
+    "f = plot(df.index, df.precipitation_rate, df.accumulated_rainfall, df.soiling)\n",
+    "\n",
+    "print(f\"average soiling loss: {df.soiling.mean()}\") "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e40e524d-e69a-46e4-ad0d-1de99e345df8",
+   "metadata": {},
+   "source": [
+    "As we can see May and July have rarely seen enough rainfall to surpass our rainfall accumulation threshold and therefore soiling losses increase throughout the month, reaching a rate as high as ~6% if no cleaning is applied. Soiling rate averages 3% across the period. Lowering the cleaning threshold, like for tilted panels, will give us a lower rate: with 10mm of rain set as the threshold the average soiling rate is 1.5%."
```

Comment:
Needs some more commas for clarity.  Also, too many see/seens.  Suggest

As we can see, May and July have rarely experienced enough rainfall to surpass our rainfall accumulation threshold and therefore soiling losses increase throughout the month, reaching a rate as high as ~6% if no cleaning is applied. Soiling rate averages 3% across the period. Lowering the cleaning threshold, e.g. for more tilted panels, will give us a lower rate: with 10mm of rain set as the threshold, the average soiling rate is 1.5%.